### DISCUSSION TITLE
Add Index instead of showing id in table

### DISCUSSION THREAD

**Discussion by nepaliayush** [rappasoft/laravel-livewire-tables#2053] [Show and tell]
// Initialize the row index
    protected int $index = 1;

public function columns(): array
    {
        return [

//this display index instead of id
            Column::make("S.No")
                ->label(function ($row, Column $column) {
                    // Return and increment the index for each row
                    return $this->index++;
                }),

//hide the id
            Column::make("Id", "id")
                ->hideIf(true)
                ->sortable(),
    ];
    }

**Comment by lrljoe** [comment]
Or see #1990 which is a cleaner way to do this
